docs/framework/wcf/samples/aspnet-compatibility.md

```diff
@@ -51,12 +53,12 @@ public interface ICalculatorSession
   
  The service maintains state, using the feature, for each client as multiple service operations are called to perform a calculation. The client can retrieve the current result by calling `Result` and can clear the result to zero by calling `Clear`.  
```

Comment:
not related to your change:

formatting on this session here on GitHub diff looks a bit odd. Not sure what's causing that. 

line 35 and 62- add language identifier?